biggerquery/job.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import absolute_import
+
+
+import os
+import runpy
+import mock
+
+import tempfile
+import imp
+from types import ModuleType
+from inspect import getargspec
+from inspect import getmodule
+from unittest import TestCase
+from unittest import main
+
+from .utils import secure_create_dataflow_manager_import
+from .dataset_manager import create_dataset_manager
+create_dataflow_manager = secure_create_dataflow_manager_import()
+
+from .utils import unzip_file_and_save_outside_zip_as_tmp_file
+from .utils import AutoDeletedTmpFile
+from .configuration import DatasetConfig
+from .configuration import DataflowConfig
+
+DEFAULT_RETRY_COUNT = 3
+DEFAULT_RETRY_PAUSE_SEC = 60
+
+
+class Job(object):
+    def __init__(self,
+                 component,
+                 id=None,
+                 retry_count=DEFAULT_RETRY_COUNT,
+                 retry_pause_sec=DEFAULT_RETRY_PAUSE_SEC,
+                 **dependency_configuration):
+        self.id = id or component.__name__
+        self.component = component
+        self.dependency_configuration = dependency_configuration
+        self.retry_count = retry_count
+        self.retry_pause_sec = retry_pause_sec
+
+    def __call__(self, runtime):
```

Comment:
why?

Reply:
I will delete it(it's just another way of running a job).

Reply:
done